src/Datasource/EntityTrait.php

```diff
@@ -260,7 +260,7 @@ public function set($property, $value = null, array $options = [])
                 continue;
             }
 
-            $setter = $this->_accessor($p, 'set');
+            $setter = static::_accessor($p, 'set');
```

Comment:
(Just for the records) I was curious myself, it seems all of the called methods are static:
https://api.cakephp.org/3.4/class-Cake.I18n.DateFormatTrait.html#_diffFormatter
https://api.cakephp.org/3.4/class-Cake.Error.Debugger.html#_log
https://api.cakephp.org/3.4/class-Cake.Error.Debugger.html#_excerpt
https://api.cakephp.org/3.4/class-Cake.Error.Debugger.html#_trace
https://api.cakephp.org/3.4/class-Cake.Datasource.EntityTrait.html#__accessor
https://api.cakephp.org/3.4/class-Cake.Error.BaseErrorHandler.html#_mapErrorCode
https://api.cakephp.org/3.4/class-Cake.ORM.Table.html#_defaultConnectionName
https://api.cakephp.org/chronos/1.0/class-Cake.Chronos.Traits.TestingAidTrait.html#_hasTestNow
https://api.cakephp.org/chronos/1.0/class-Cake.Chronos.Traits.TestingAidTrait.html#_getTestNow